functions/ssl_logging.pp

```diff
@@ -0,0 +1,24 @@
+function nrpe::ssl_logging(
+  Boolean $ssl_log_startup_params,
+  Boolean $ssl_log_remote_ip,
+  Boolean $ssl_log_protocol_version,
+  Boolean $ssl_log_cipher,
+  Boolean $ssl_log_client_cert,
+  Boolean $ssl_log_client_cert_details,
+) >> String {
```

Comment:
very interesting syntax

Reply:
could you add a comment to the function and describe the purpose?

Reply:
Done.  I could have built the string inside the `epp`, but decided to pull it into a separate function.

Reply:
I think you can use regular rspec function tests on them as well.

Reply:
Yes, it seemed easier to test it that way.